### ISSUE TITLE
Pre-training hedgedocs

### ISSUE THREAD

**Issue by shmh40** [ecmwf/WeatherGenerator#2309]
A collection of hedgedocs relevant to WeatherGenerator pre-training.

1. [Master pre-training comparison](https://gitlab.jsc.fz-juelich.de/hedgedoc/0xoxOW98Tu2WSXwHOADtEA)
2. [JEPA pre-training with physical decoder regularisation](https://gitlab.jsc.fz-juelich.de/hedgedoc/J04ESQNHTtau_2RiHhMaOw) 
4. [Comparison of temporal, spatial JEPA with forecast model](https://gitlab.jsc.fz-juelich.de/hedgedoc/R7goHa63Rwi3cfP7EZ7H7g)
5. [Temporal JEPA with different heads ablation](https://gitlab.jsc.fz-juelich.de/hedgedoc/3cHrRoBIT7KcXfBduK9tmg)
6. [MTM vs JEPA vs forecast pre-training comparison](https://gitlab.jsc.fz-juelich.de/hedgedoc/7ooap9dWStSpayFoq38WTg)
7. [JEPA N320 ERA5 experiments](https://gitlab.jsc.fz-juelich.de/hedgedoc/EEkhuYPxQcyi5Aw61gP4Wg)
8. [MTM latent dimension ablation](https://gitlab.jsc.fz-juelich.de/hedgedoc/wARvvd5YTJiVa6uDxn8UmA)
9. [Spatial vs. temporal JEPA comparison](https://gitlab.jsc.fz-juelich.de/hedgedoc/TWgY7mggQd2vQKFa22qkwA)
10. [Latent forecast JEPA and LeJEPA comparison, April 2026](https://gitlab.jsc.fz-juelich.de/hedgedoc/HFrwAtySS-abJLwb6cgfxA)
11. [Spatial JEPA prediction head ablation](https://gitlab.jsc.fz-juelich.de/hedgedoc/MWh3ATNNTmypK9hKsPrzwQ)
12. [V-JEPA 2.1 first implementation experiments 3-step ERA5 finetune](https://gitlab.jsc.fz-juelich.de/hedgedoc/MQmuQLDGRB-HKSGwrP8D-A)
13. [Longer spatial JEPA pre-training](https://gitlab.jsc.fz-juelich.de/hedgedoc/JAOJaRW7S2O3pFmpAilhAw)
14. [JEPA EMA ablation](https://gitlab.jsc.fz-juelich.de/hedgedoc/GN4HrQfVTF2kG8QcPZCzOA)
15. [MTM pre-training experiment setups](https://gitlab.jsc.fz-juelich.de/hedgedoc/GTri2WZTQzq3urXemPpbqQ)
16. [MAE vs. WeatherGenerator masked token modelling](https://gitlab.jsc.fz-juelich.de/hedgedoc/l1WvK72qR_OqVx-Onb2V9w)
17. [MTM pre-training method comparison with forecast model](https://gitlab.jsc.fz-juelich.de/hedgedoc/lcSrFlV-QmygCV15eZ7vRw)
18. [Multi-dataset MTM training](https://gitlab.jsc.fz-juelich.de/hedgedoc/yORDs0xQQoK9EbhWO2vhpA)
19. [JEPA pre-training planning document March 2026](https://gitlab.jsc.fz-juelich.de/hedgedoc/jDYPnydERD66tgPriuXNdw)
20. [Multiple obs dataset pre-training](https://gitlab.jsc.fz-juelich.de/hedgedoc/JYuEjAjwSJy0r590LEBmQQ)
21. [JEPA healpix-level masking ablation](https://gitlab.jsc.fz-juelich.de/hedgedoc/cYYLUF2sTgWak00BL3g35w)
22. [Frozen teacher JEPA spatial experiments](https://gitlab.jsc.fz-juelich.de/hedgedoc/0Gk7M3MrQAqVKXKw_wE-9g)
23. [JEPA cropping experiments](https://gitlab.jsc.fz-juelich.de/hedgedoc/QReDqI5zSkKO3MmadU9ZvA)
24. [Autocorrelation lengths per-channel](https://gitlab.jsc.fz-juelich.de/hedgedoc/IrWg_M8aSrCCjjXJI0uRzg)
25. [Frozen JEPA, 2D Rope, learning rate and masking rate ablation](https://gitlab.jsc.fz-juelich.de/hedgedoc/OgGOfs2-RVOZcEEjB0108w)
26. [Ablation of predictor network in frozen teacher JEPA, March 2026](https://gitlab.jsc.fz-juelich.de/hedgedoc/EScRViRyTEiK8trgA9f4IQ)
27. [Exploratory MTM experiments with healpix-level masking matched to variable autocorrelation](https://gitlab.jsc.fz-juelich.de/hedgedoc/BnmlrucxTWWuBq0p_LVBrw)
28. [JEPA pre-training, NPP-ATMS finetuning, March 2026](https://gitlab.jsc.fz-juelich.de/hedgedoc/-4Wl7hrBRQ-WjDV7hwJYKw)
29. [Setup for first iteration of JEPA experiments](https://gitlab.jsc.fz-juelich.de/hedgedoc/DiGQJ00sQVeW6-06ExOlow)
30. [Masking visualisation](https://gitlab.jsc.fz-juelich.de/hedgedoc/OQXH8V5yRZOy7oJvbnvgAA)
31. [Masked pretraining - 9 step finetuning](https://gitlab.jsc.fz-juelich.de/hedgedoc/zD1qBRqBThqRf2k-aJXFjg)
32. [Masked token modeling pretraining](https://gitlab.jsc.fz-juelich.de/hedgedoc/7ooap9dWStSpayFoq38WTg)
33. [MTM Evaluation - Global Engine Ablation](https://gitlab.jsc.fz-juelich.de/hedgedoc/idiIRi57Q1eoblrhLK-CjQ)

**Comment by shmh40** [comment]
@sophie-xhonneux @wael-mika @csjfwang @Sindhu-Vasireddy @iluise @clessig  

Please feel free to edit/update descriptions etc.
